### ISSUE TITLE
Audio Quality

### ISSUE THREAD

**Issue by trinib** [nativefier/nativefier#1488]
### Homework

- [X] I took the time to write a good, descriptive issue title
- [X] I read `nativefier --help` and [API.md](https://github.com/nativefier/nativefier/blob/master/API.md).
- [X] I checked [CATALOG.md](https://github.com/nativefier/nativefier/blob/master/CATALOG.md) for community suggestions & workarounds.
- [X] I searched [existing issues, open & closed](https://github.com/nativefier/nativefier/issues?q=is%3Aissue). Yes, my question is new.
- [X] I'm running the [latest version](https://github.com/nativefier/nativefier/releases).

### Your question

Is it normal for electron apps to use some type of builtin audio? On Windows I do not get the high quality sound I would get from browser. I use Realtek Audio Console and Sonic Studio 3.  Both does detect YouTube app built with nativefier with regular volume mixer but the sound effects with SS3 and DTS with RAC do not work.

### Nativefier version

50.0.1

### Node.js version

18.12.1

### npm version

8.19.2

### OS

21H2 (OS Build 19044.1566)

**Comment by TheCleric** [comment]
I believe Electron should be leaving this up to the OS/Realtek to determine. I don't know why your app would be excluded from these. Are there any other apps that get excluded from these features? I'm wondering if Realtek just has a whitelist of apps it supports.

**Comment by trinib** [comment]
yes its weird that the sound effects part do not work for both Realtek Audio Console and Sonic Studio 3 but can only control the volume from them 🤷‍♂️
